auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,84 @@
+              <p v-if="typeof item === 'string'">{{ item }}</p>
+              <p v-else v-html="item.content"></p>
+            </li>
+          </ul>
+        </li>
+      </ol>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      helpContent: {
+        'title': 'Help with Starting a Business',
+        'intro': 'Start a named or numbered business in B.C. by following these steps:',
+        'steps': [
+          {
+            'title': 'Decide on a Business Type',
+            'items': [
+              {
+                'content': "Decide which business structure is the most appropriate for you. A few options are: a sole proprietorship, partnership, or corporation. Each structure has different legal and financial implications. <a href='https://entity-selection-prod.apps.silver.devops.gov.bc.ca/'>Use the Business Structures Wizard to help you decide.</a>",
+                'isHtml': true
+              },
+              'If you want to start a corporation, you also have the choice of using a named or numbered company.'
+            ]
+          },
+          {
+            'title': 'Request a Business Name or Use a Numbered Company',
+            'items': [
+              'Both named and numbered companies are started by selecting the "Get Started with a B.C.-based business button.',
+              'If you would like to start a named business you must first submit a name request. Create a unique name for your business and submit this name for examination by the Business Registry.'
+            ]
+          },
+          {
+            'title': 'Incorporate or Register',
+            'items': [
+              'If you requested your business name through your account, your Name Request (NR) will appear in your Business Registry dashboard. If you did not request your name through your account, you will need to add your Name Request to your dashboard manually using the name your requested or your NR number. You can track the approval status of your name request on your Business Registry dashboard.',
+              'If one of your requested names is approved, you can use it to incorporate or register your business by selecting "Register Now" beside the name.',
+              'Follow the steps in the application and complete all of the required information including addresses, contact information, people and roles, and share structure (when applicable).',
+              "Retain a copy of all incorporation or registration documents for your business' records."
+            ]
+          },
+          {
+            'title': 'Manage and Maintain Your Business',
+            'items': [
+              'Once your business is incorporated or registered you are required to keep information about your business up to date with the Business Registry.',
+              'By managing your business through your BC Registries account you can:',
+              'View and change business information.',
```

Comment:
Still, this line and the 3 below it should be indented (sub-bullets), as per uxpin:

![image](https://github.com/bcgov/sbc-auth/assets/10002889/c62895b4-42a7-4e48-9e71-6ee2b5f2e2a0)

Reply:
Change should be in next commit, I wanted to test the general component first